ui/00_message/pawn/p0003.xml

```diff
@@ -13,8 +13,7 @@ Master, please give me a chance!</translation>
   <message>
     <original>覚者様！　どうか我がマスターに！
 私に活躍の機会を！</original>
-    <translation>Arisen! Please be my master! 
-Give me an opportunity to flourish!</translation>
+    <translation>Arisen! Please be my master! Give me an opportunity to flourish!</translation>
   </message>
   <message>
     <original>忠誠の心なら、誰にも負けません！</original>
```

Comment:
I feel like this is an instance where the 1-to-1 translation sounds odd in English, but this is just my opinion:
"If it's a matter of loyalty, I will not lose to anyone!